utilities/list-helpers.metta

```diff
@@ -207,7 +207,7 @@
 
 (: setUnion (-> (List $t) (List $t) (List $t) (List $t)))
```

Comment:
It's better if we refactor the `setUnion` function like the `setDifference` function for the sake of consistency

Reply:
I've refactored it.